utils/zerovec/src/ule/plain.rs

```diff
@@ -59,25 +60,26 @@ macro_rules! impl_byte_slice_size {
                 unsafe { core::slice::from_raw_parts_mut(data as *mut Self, len) }
             }
 
-            /// Gets this RawBytesULE as an unsigned int. This is equivalent to calling
-            /// [AsULE::from_unaligned()] on the appropriately sized type.
+            /// Gets this `RawBytesULE` as a `$unsigned`. This is equivalent to calling
+            /// [`AsULE::from_unaligned()`] on the appropriately sized type.
             #[inline]
             pub fn as_unsigned_int(&self) -> $unsigned {
                 <$unsigned as $crate::ule::AsULE>::from_unaligned(*self)
             }
 
-            /// Convert an array of native-endian aligned integers to an array of RawBytesULE.
-            pub const fn from_array<const N: usize>(arr: [$unsigned; N]) -> [Self; N] {
-                let mut result = [RawBytesULE([0; $size]); N];
-                let mut i = 0;
-                // Won't panic because i < N and arr has length N
-                #[allow(clippy::indexing_slicing)]
-                while i < N {
-                    result[i].0 = arr[i].to_le_bytes();
-                    i += 1;
-                }
-                result
+            /// Convert a `$unsigned` to `RawBytesULE`. This is equivalent to calling
```

Comment:
nit: to generate correct docs, you need to do something like
```suggestion
            #[doc(concat!("Convert a `", stringify!($unsigned) ,"` to `RawBytesULE`. This is equivalent to calling")]
```